tests/workflows/test_tsc_ic_usd.py

```diff
@@ -0,0 +1,49 @@
+import os
+from argparse import Namespace
+
+import pandas as pd
+
+from mawpy.constants import (USER_ID, UNIX_START_T, UNIX_START_DATE, ORIG_LAT, ORIG_LONG, ORIG_UNC, STAY_LAT, STAY_LONG,
+                             STAY, STAY_DUR)
+from mawpy.workflows.tsc_ic_usd import tsc_ic_usd
+
+
+def test_tsc_ic_usd(tmp_path):
+
+    input_file = os.path.dirname(__file__) + '/../resources/test_input.csv'
+    output_file = os.path.join(tmp_path, 'test_output_workflow_tsc_ic_usd.csv')
+    # Prepare arguments similar to command-line arguments
+    args = Namespace(
+        input_file=input_file,
+        output_file=output_file,
+        spatial_constraint_1=1,
+        spatial_constraint_2=1,
+        duration_constraint_1=300,
+        duration_constraint_2=0,
+        duration_constraint_3=300
+    )
```

Comment:
I'm still not conviced this is better than just doing :) :
```
   actual_output_df = tsc_ic_usd(input_file, output_file, 1, 2, 300, 0, 300)
```
what are we testing with the Namespace that we are not with just the values? The code is much more concise.

Reply:
Updated for this one, will update for others as well in the following PRs.